concepts/admission-webhook-cert-management/README.md

```diff
@@ -0,0 +1,145 @@
+
+https://github.com/gardener/cert-management
+
+Since Kyma-on-Gardener is a very common setup, and Gardener has a pre-installed cert-manager, it seems to be a natural choice. However Gardener `cert-manager` is actually a separate open source project and is very different from the upstrem `cert-manager`. It comes with a different feature set (it even has its own CRDs). From the Github project description:
+
+>In a multi-cluster environment like Gardener, using existing open source projects for certificate management like cert-manager becomes cumbersome. With this project the separation of concerns between multiple clusters is realized more easily. The cert-controller-manager runs in a secured cluster where the issuer secrets are stored. At the same time it watches an untrusted source cluster and can provide certificates for it.
+
+However, according to the project maintainers the Gardener `cert-manager` is built for mainly for the “Let’s Encrypt” use-cases (publicly resolvable domain names). It does not support self-signed issuers, nor it supports caBundle injection. The project maintainers suggest using the upstream `cert-manager` from the community for self-signed/webhook certs.
+
+Moreover, there are other supported non-Gardener Kyma environments (k3d), where the Gardener `cert-manager` is not pre-installed. 
+### <a name="upstream-cert-manager"></a>Upstream cert-manager
+
+https://github.com/cert-manager/cert-manager
+
+<img src="assets/admission-controller-certs-with-cert-manager.drawio.svg">
+
+The `cert-manager` is an operator running inside a Kyma cluster. It is a Kyma component itself and is used by other Kyma components that have custom webhook servers. A shared self-signed `Issuer` is also deployed.
+  ```yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+  namespace: sandbox
+spec:
+  selfSigned: {}
+```
+
+Each component that needs a TLS certificate would just deploy a `Certificate` resource and mount the generated secret as files to be used by the server container. Since all Kyma operators are implemented using Kubebuilder, secret rotation will work out of the box: whenever the `cert-manager` rotates a secret, the server will notice the files change and reload the http handler. 
+  ```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: serving-cert
+  namespace: kyma-system
+spec:
+  # $(SERVICE_NAME) and $(SERVICE_NAMESPACE) must be substituted by the webhook service/namespace
+  dnsNames:
+  - $(SERVICE_NAME).$(SERVICE_NAMESPACE).svc
+  - $(SERVICE_NAME).$(SERVICE_NAMESPACE).svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: selfsigned-issuer
+  secretName: webhook-server-cert
+```
+
+In addition to that, it would mark it's webhook configuration with a special anntotation to let the `ca injector` automatically populate the `caBundle`. See [more](https://cert-manager.io/docs/concepts/ca-injector/).
```

Comment:
```suggestion
In addition to that, it would mark its webhook configuration with a special annotation to let the `ca injector` automatically populate the `caBundle`. See [CA Injector](https://cert-manager.io/docs/concepts/ca-injector/).
```